### PR TITLE
AxisCommunications.NetEye200 version latest

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108036)

### CODE REVIEW

**Comment by stephengillie** [comment]
This package appears to be a self-extracting zip file.

File: manifests/a/AxisCommunications/NetEye200/latest/AxisCommunications.NetEye200.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: AxisCommunications.NetEye200
+PackageVersion: latest
```
Comment (Trenly) [inline manifests/a/AxisCommunications/NetEye200/latest/AxisCommunications.NetEye200.installer.yaml:5]:
```suggestion
PackageVersion: 1.32
```
`latest` will now allow winget to understand if the package can be upgraded or not. Please change this to a semantic version. 

image

File: manifests/a/AxisCommunications/NetEye200/latest/AxisCommunications.NetEye200.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AxisCommunications.NetEye200
+PackageVersion: latest
```
Comment (Trenly) [inline manifests/a/AxisCommunications/NetEye200/latest/AxisCommunications.NetEye200.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.32
```

File: manifests/a/AxisCommunications/NetEye200/latest/AxisCommunications.NetEye200.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: AxisCommunications.NetEye200
+PackageVersion: latest
```
Comment (Trenly) [inline manifests/a/AxisCommunications/NetEye200/latest/AxisCommunications.NetEye200.yaml:5]:
```suggestion
PackageVersion: 1.32
```

**Review by Trenly — CHANGES_REQUESTED** [review]
As Stephen said - this appears to be a self-extracting Zip, which is something winget is not able to handle and which does not write any ARP entries, so winget would not be able to detect it even if it is installed

**Comment by stephengillie** [comment]
Maybe it needs a `NestedInstallerType` and similar settings? Running the symlink still launches the self-extracting Zip: 
image

**Comment by upintheairsheep** [comment]
> Maybe it needs a `NestedInstallerType` and similar settings? Running the symlink still launches the self-extracting Zip: image

**Comment by stephengillie** [comment]
>  Manifest is invalid. (Line: 11, Col: 11, Idx: 452) - (Line: 11, Col: 11, Idx: 452): While scanning a plain scalar value, found invalid mapping.

File: manifests/a/AxisCommunications/NetEye200/1.32/AxisCommunications.NetEye200.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: AxisCommunications.NetEye200
+PackageVersion: 1.32
+Installers:
+- InstallerUrl: http://ftp.axis.com/pub/axis/software/cam_srv/neteye_200/latest/axis200.exe
+  Architecture: x86
+  InstallerType: portable
+  InstallerSha256: 3B26B848290AE3239153037F6F3FC9F6E0E7F5299D9290D5EE31140FF1A39F8A
+    Silent: /auto %USERPROFILE%\AppData\Roaming\Axis\NetEye200
```
Comment (stephengillie) [inline manifests/a/AxisCommunications/NetEye200/1.32/AxisCommunications.NetEye200.installer.yaml:11]:
```suggestion
  InstallerSwitches:
    Silent: /auto %USERPROFILE%\AppData\Roaming\Axis\NetEye200
```

Comment (stephengillie) [reply manifests/a/AxisCommunications/NetEye200/1.32/AxisCommunications.NetEye200.installer.yaml:11]: This allows the install to be successful, but unfortunately doesn't silently run the self-extracting Zip file.

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Pull request closed without merging**